tests/e2e/performance/performance.spec.js

```diff
@@ -1,18 +1,25 @@
+/* eslint-disable capitalized-comments */
 import { test, expect } from 'playwright-test-coverage'
+import twentyCollections from './__mocks__/twenty_collections.json'
 
 test.describe('Performance Benchmarking', () => {
   test.beforeEach(async ({ page }) => {
-    await page.route('**/*.{png,jpg,jpeg}', (route) => route.abort())
```

Comment:
This is just to mock the map so we're making gibs call here aren't those going to mess with the metric values?

Reply:
I reverted this change so that all GIBS requests get aborted again.